src/requests.js

```diff
@@ -20,28 +21,78 @@ function safeHeader(safe, last_modified) {
   return {"If-None-Match": "*"};
 }
 
-/**
- * @private
- */
-export function createBucket(bucketName, options={}) {
-  if (!bucketName) {
-    throw new Error("A bucket name is required.");
-  }
-  // Note that we simply ignore any "bucket" option passed here, as the one
-  // we're interested in is the one provided as a required argument.
-  const { data={}, headers, permissions, safe } = {
+
+function _create(path, {data, permissions}, options={}) {
```

Comment:
Nit: as we're already in a module and these functions are not exported, it's probably not necessary to prefix them with an underscore.